Rakefile.rb

```diff
@@ -34,12 +34,50 @@
     post.puts "modified: #{Time.now.strftime('%Y-%m-%d %H:%M:%S %z')}"
     post.puts "description: \"Some description goes here.\""
     post.puts "tags: [#{tags}]"
+    post.puts "featured_post: false"
     post.puts "comments: true"
     post.puts "share: true"
+    post.puts "featured_post: false"
     post.puts "---"
   end
 end
 
+# usage rake featured_post
```

Comment:
Thanks for this Rake task! Ideally, this comment should be in README.

Reply:
Ok. I've added.